### PR TITLE
cloud: document preprod and new CD process

### PR DESCRIPTION
**Pull request by danieldides**
This PR aims to start scoping out the preprod release process. These pages are the supporting evidence for SOC2 - GN-100 - Sourcegraph maintains separate development, test, and production environments.

Some of the work to make these documents true is not completed yet. This is just the first pass.

### CODE REVIEW

File: content/departments/product-engineering/engineering/cloud/devops/index.md [hunk 1]
```diff
@@ -28,7 +28,10 @@ Our Cloud monitoring infrastructure (Prometheus / Grafana)
 ## Content
 
 - [How to deploy a code change to the Cloud](deploy-code-change.md)
-- [Large release (rollout release) process](deploy-code-change.md#large-releases-to-cloud-rollup-releases)
+  <<<<<<< Updated upstream
+- # [Large release (rollout release) process](deploy-code-change.md#large-releases-to-cloud-rollup-releases)
+- [Large-release-(rollout-release)-process](<deploy-code-change.md#Large-releases-to-Cloud-(Rollup-releases)>)
+  > > > > > > > Stashed changes
```
Comment (daxmc99) [inline content/departments/product-engineering/engineering/cloud/devops/index.md]:
```suggestion

```

File: content/departments/product-engineering/engineering/cloud/devops/index.md [hunk 2]
```diff
@@ -28,7 +28,10 @@ Our Cloud monitoring infrastructure (Prometheus / Grafana)
 ## Content
 
 - [How to deploy a code change to the Cloud](deploy-code-change.md)
-- [Large release (rollout release) process](deploy-code-change.md#large-releases-to-cloud-rollup-releases)
+  <<<<<<< Updated upstream
```
Comment (daxmc99) [inline content/departments/product-engineering/engineering/cloud/devops/index.md]:
```suggestion
```

File: content/departments/product-engineering/engineering/cloud/devops/preprod.md [hunk 3]
```diff
@@ -1,8 +1,13 @@
 # Cloud preprod environment (aka staging)
 
+Preprod is live and deployed at https://preivew.sgdev.dev.
```
Comment (sanderginn) [inline content/departments/product-engineering/engineering/cloud/devops/preprod.md]:
```suggestion
Preprod is live and deployed at https://preview.sgdev.dev.
```

**Review by sanderginn — COMMENTED** [review]
Small typo, for the rest 👍

**Pull request merged**